blockly-ws-search/workspace_search.js

```diff
@@ -165,6 +183,21 @@ class WorkspaceSearch {
     }
   }
 
+    /**
+   * Add listener on the workspace to open the search bar when Control F or
+   * Command F are used.
+   * TODO: We might want Blockly to be able to deal with setting shortcuts on
+   * workspaces.
+   * @param {KeyboardEvent} e The key down event.
+   * @private
+   */
+  onWorkspaceKeyDown_ = function(e) {
+    if ((e.ctrlKey || e.metaKey) && e.keyCode == Blockly.utils.KeyCodes.F) {
```

Comment:
The keyCode property is deprecated (on KeyboardEvent) so I think its recommended to use the key or code property instead (I noticed because my IDE gave warning for using keyCode). I'm not certain what the right approach is, it probably depends on what browsers we want to support. Currently I'm using key in onKeyDown_.